e2e/test/iothub/service/RegistryManagerE2ETests.cs

```diff
@@ -18,8 +18,7 @@ namespace Microsoft.Azure.Devices.E2ETests.Iothub.Service
     [TestCategory("IoTHub")]
     public class RegistryManagerE2ETests : E2EMsTestBase
     {
-        private readonly string _devicePrefix = $"E2E_{nameof(RegistryManagerE2ETests)}_";
```

Comment:
As they are the same, just generalize it. I wanted to use it for configurations, too.